CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Changes
 19.09 to 20.01
 --------------
 * [Developer]: Upgraded development Java version to Java 11 (LTS).
+* [UI]: Added the ability to delete multiple analyses from both the Project > Analyses and List All Analyses pages.
+* [UI]: Updated project and global analyses listing pages to use Ant Design tables.
```

Comment:
The Sequencing Runs table was also updated so we should add it in here

Reply:
Updated in 5dfe3c3ed5baf6fdc962abff1f3cc164f557c73a